src/transform.ts

```diff
@@ -212,6 +212,26 @@ export interface WindowTransform {
   sort?: SortField[];
 }
 
+export interface ImputeSequence {
+  /**
+   * The starting value of the sequence.
+   */
+  start: number;
```

Comment:
I believe that Vega expression's sequence function supports a sequence with only `stop` and will start with `0` (or `1`). 

I wonder if we should support sequence with only `stop` (akin to python's `xrange()`). 

If you agree, please update the docs and modify the behavior. 
 (It's okay to always call sequence with 3 params.)

Reply:
I think it makes sense. Should I check for the case when `stop` and `step` are specified but `start` is not or maybe infer a default?


Also, I have set a default for `step` in my code which I will remove, since that should be dealt with by Vega.

Reply:
```
const {start, stop, step} = keyvals;
const result = [
    ...(start ? [start] : []),
    stop,
    ...(step ? [step] : [])
].join(',');
return {signal: `sequence(${result})`};
 ```
Is this okay?

Reply:
That would be wrong when stop and step are specified.  

I think it's actually better to to always call Vega expression's `sequence` with 3 params.